raiden/tests/unit/blockchain/test_events.py

```diff
@@ -0,0 +1,73 @@
+from unittest.mock import Mock
+
+from hexbytes import HexBytes
+
+from raiden.blockchain.events import BlockchainEvents
+from raiden.tests.utils.events import check_dict_nested_attrs
+from raiden.utils.filters import StatelessFilter
+
+
+def stub_web3(logs):
```

Comment:
why call this stub and not mock?

Reply:
We do nothing with it except feed some test data to the testee, so stub is the more accurate term.